### PR TITLE
V11 migration

### PR DESCRIPTION
**Pull request by Alchez**

### CODE REVIEW

File: repairs/public/js/work_order.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-erpnext.production_order.make_se = function (frm, purpose) {
+erpnext.work_order.make_se = function (frm, purpose) {
```
Comment (vjFaLk) [inline repairs/public/js/work_order.js]: Codacy Issue found: ['erpnext' is not defined. (no-undef)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)

Comment (vjFaLk) [inline repairs/public/js/work_order.js]: Codacy Issue found: [Identifier 'make_se' is not in camel case. (camelcase)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)

File: repairs/public/js/warranty_claim.js [hunk 2]
```diff
@@ -73,7 +73,21 @@ frappe.ui.form.on("Warranty Claim", {
 
 			// Start testing the item
 			if (frm.doc.status == "To Test") {
-				var repair_btn = frm.add_custom_button(__("Test Item"), () => {
+				let requires_scan = {
+					left_ear: false,
+					right_ear: false
```
Comment (vjFaLk) [inline repairs/public/js/warranty_claim.js]: Codacy Issue found: [Identifier 'right_ear' is not in camel case. (camelcase)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)

File: repairs/public/js/warranty_claim.js [hunk 3]
```diff
@@ -73,7 +73,21 @@ frappe.ui.form.on("Warranty Claim", {
 
 			// Start testing the item
 			if (frm.doc.status == "To Test") {
-				var repair_btn = frm.add_custom_button(__("Test Item"), () => {
+				let requires_scan = {
+					left_ear: false,
```
Comment (vjFaLk) [inline repairs/public/js/warranty_claim.js]: Codacy Issue found: [Identifier 'left_ear' is not in camel case. (camelcase)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)

File: repairs/public/js/warranty_claim.js (hunk 2 above)
Comment (vjFaLk) [inline repairs/public/js/warranty_claim.js:78]: Codacy Issue found: [Identifier 'right_ear' is not in camel case. (camelcase)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)

File: repairs/public/js/warranty_claim.js (hunk 3 above)
Comment (vjFaLk) [inline repairs/public/js/warranty_claim.js:77]: Codacy Issue found: [Identifier 'left_ear' is not in camel case. (camelcase)](https://app.codacy.com/manual/DigiThinkIT/repairs/pullRequest?prid=4737428)

**Review by DeveloperMetal — APPROVED** [review]

**Pull request merged**